Extension Project_Final.Rmd

```diff
@@ -0,0 +1,768 @@
+```
+
+
+
+*1 Model for Hypothesis 5:* 
+(5) remote mountainous areas will have higher rates of violence due to the absence of state institutions
+
+
+```{r, warning=FALSE, error=FALSE}
+Three <- glm.nb(hom.rate ~ weigh.elev + pol1k + pct.ej + pct.com + pct.individual + pct.coffee + pct.cattle + gini + log.corn.yield + log.pop.dens + pct.young + pct.indi + HDI + pct.fem.house + docs_10k + dummy.SOM, data = sample) 
+```
+
+```{r, results = 'asis'}
+gsub("\n[[:space:]]","", htmlreg(extract.negbin(Three), caption = "Table 3: Coefficients from the Negative Binomial Regression Models of Homicide on State Presence (Hypothesis 5)", caption.above = TRUE, stars = c(.05, .01), custom.coef.names = c("Constant", "SD of altitude", "Police per 1,000", "% ejido surface area", "% communal surface area", "% individual production", "% coffee production", "% cattle production","Gini index", "Log maize yields", "Log population density", "% young males", "% indigenous", "Human Development Index", "% female-headed households", "Medical doctors per 10,000", "Valley of Mexico"), custom.note = "SEs in parentheses,         * P < .05, ** P < .01, two-tailed tests", include.intercept = TRUE, reorder.coef = c(2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 1), groups = list("State presence" = 1:2, "Collective ownership" = 3:4, "Organization of agricultural production" =5:7, "Plot size and crop yields" = 8:9, "Demographic variables" = 10:12, "Resource deprivation" = 13, "Family structure" = 14, "Access to medical attention" = 15, "Geographical region" = 16), inline.css = TRUE, doctype = FALSE, html.tag = FALSE, head.tag = FALSE, body.tag = FALSE, center = FALSE, digits = 4, star.symbol = "\\*")) 
+```
+
+
+
+**Conclusions**
+Our extension shows that (as expected) poverty, inequality, the percentage of young men in a municipality, and the percentage of female-headed households in a municipality are related to the municipality's homicide rate. The municipality's degree of collective ownership is also related to its homicide rate. However, with 2007-10 data, Villarreal's main conclusion that state presence (measured by altitude and an additional variable, police per 1,000 people) no longer holds. The coefficient for weighted SD of elevation is barely significant, with an SE half its size. 
```

Comment:
Excellent job overall. You did an impressive amount of diligent data cleaning and very nice graphs!